circle.yml

```diff
@@ -134,6 +154,18 @@ step-library:
       store_artifacts:
         path: mapbox-gl-js/test/integration/render-tests/index-recycle-map.html
         destination: render-tests
+  - &upload-ios-build-log
+      store_artifacts:
+        path: ~/Library/Logs/scan
+        destination: scan-logs-build
+  - &upload-ios-test-log
+      store_artifacts:
+        path: ~/Library/Logs/scan
```

Comment:
This is the same path as above?

Reply:
The build and test steps are separated, but output logs to the same path with the same name. There’s not a mechanism have the files named differently (and the second would overwrite the first), so they’re copied from the same source folder to differently named artifacts folders.

Reply:
I agree that this is inelegant — maybe having a step that renamed the logs would be better.